### ISSUE TITLE
runtime.connect only accepts two arguments

### ISSUE THREAD

**Issue by arty-name** [Lusito/webextension-polyfill-ts#59]
According to [the MDN docs for runtime.connect](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/connect) the first parameter `extensionId` is optional, and there is even an example of calling this function with the "second" parameter only: 
```js
var myPort = browser.runtime.connect({name:"port-from-cs"});
```
The current types however prevent omitting the first parameter while providing the second one (`Argument of type '{ name: string; }' is not assignable to parameter of type 'string'.`). Could you please change that?

**Comment by Lusito** [comment]
Thanks for the report. Fixed in 0.26.0.

**Comment by arty-name** [comment]
Thanks!
